src/graph/GoExecutor.cpp

```diff
@@ -657,21 +673,35 @@ void GoExecutor::processFinalResult(RpcResponse &rpcResp, Callback cb) const {
             RowSetReader rsReader(eschema, vdata.edge_data);
             auto iter = rsReader.begin();
             while (iter) {
```

Comment:
I do not understand why not judge the condition “iter != rsRead.end()” in the “while”?